modules/RTC/ortc/RTCSessionDescription.js

```diff
@@ -0,0 +1,86 @@
+import sdpTransform from 'sdp-transform';
+
+/**
+ * RTCSessionDescription implementation.
+ */
+export default class RTCSessionDescription {
+    /**
+     * RTCSessionDescription constructor.
+     * @param {object} [descriptionInitDict]
+     * @param {string} [descriptionInitDict.type] "offer" / "answer".
+     * @param {string|object} [descriptionInitDict.sdp] SDP string or
```

Comment:
What I don't like is that we will never pass sdp-transform object to `RTCSessionDescription` because it doesn't support it. Except this shim.